tests/env_variables/test_environment_variables.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pytest
+import requests
+
+HUGGING_FACE_API_URL = "https://huggingface.co/api/whoami"
+GITHUB_API_URL = "https://api.github.com/user"
+
+do_not_run_in_debug = lambda: os.getenv("FLASK_DEBUG") == "1"
+
+
+class TestEnvironmentVariables:
+    """Tests for expiring environment variables."""
+
+    @pytest.mark.xfail(
+        condition=do_not_run_in_debug,
+        reason="Skipping in debug mode",
+        run=False,
+    )
```

Comment:
```suggestion
    @pytest.mark.skipif(
        do_not_run_in_debug(),
        reason="Skipping in debug mode",
    )
```
__suggestion, blocking:__ Alter this and the below test to use this format

Apologies for not catching this before. Unfortunately, tests marked with `xfail` with `run` set to `False` will just never run, even in deployment. You'll notice this in the tests running for this PR, where your tests were just immediately x-failed. The `skipif` mark is a little bit easier to work with, so I'd recommend changing to this.

Reply:
modified file with skipif